android/src/main/java/com/adyenreactnativesdk/component/dropin/DropInModule.kt

```diff
@@ -195,6 +200,57 @@ class DropInModule(context: ReactApplicationContext?) : BaseModule(context), Rea
         CheckoutProxy.shared.advancedService?.sendRecurringResult(result)
     }
 
+    @ReactMethod
+    fun provideBalance(success: Boolean, balance: ReadableMap?, error: ReadableMap?) {
+        val listener = getService()
+        if (listener == null) {
+            sendErrorEvent(ModuleException.NoModuleListener(integration))
+            return
+        }
+        if (success) {
+            val jsonObject = ReactNativeJson.convertMapToJson(balance)
+            val balanceResult = BalanceResult.SERIALIZER.deserialize(jsonObject)
+            listener.sendBalanceResult(BalanceDropInServiceResult.Balance(balanceResult))
+        } else {
+            val message = error?.getString(AdyenConstants.PARAMETER_MESSAGE)
+            listener.sendBalanceResult(BalanceDropInServiceResult.Error(null, message, true)  )
+        }
+    }
+
+    @ReactMethod
+    fun provideOrder(success: Boolean, order: ReadableMap?, error: ReadableMap?) {
+        val listener = getService()
+        if (listener == null) {
+            sendErrorEvent(ModuleException.NoModuleListener(integration))
+            return
+        }
+        if (success) {
+            val jsonObject = ReactNativeJson.convertMapToJson(order)
+            val orderResponse = OrderResponse.SERIALIZER.deserialize(jsonObject)
+            listener.sendOrderResult(OrderDropInServiceResult.OrderCreated(orderResponse))
+        } else {
+            val message = error?.getString(AdyenConstants.PARAMETER_MESSAGE)
+            listener.sendOrderResult(OrderDropInServiceResult.Error(null, message, true)  )
```

Comment:
Also the last parameter here is `dismissDropIn` which is by default `false`. Is it intentionally set to `true` here? That will dismiss drop-in automatically, either immediately or if a pop-up is shown after the dismissal of that. If it's not intentional I suggest not passing it so Android SDK would decide what to do.

Reply:
Related to https://github.com/Adyen/adyen-react-native/pull/542#discussion_r1815057166